Mathlib/Topology/Connected/PathConnected.lean

```diff
@@ -198,14 +198,14 @@ compact-open topology on the space `C(I,X)` of continuous maps from `I` to `X`.
 instance topologicalSpace : TopologicalSpace (Path x y) :=
   TopologicalSpace.induced ((↑) : _ → C(I, X)) ContinuousMap.compactOpen
 
-theorem continuous_eval : Continuous fun p : Path x y × I => p.1 p.2 :=
-  ContinuousMap.continuous_eval.comp <|
-    (continuous_induced_dom (α := Path x y)).prod_map continuous_id
+instance : ContinuousEval (Path x y) I X := .of_continuous_forget continuous_induced_dom
 
-@[continuity]
+@[deprecated (since := "2024-10-04")] protected alias continuous_eval := continuous_eval
```

Comment:
I think protecting the alias defeats the point of the deprecation mechanism : the goal is that any code using this would still compile and suggest the replacement. But if unprotecting it is too annoying in that case it's not a big deal.

Reply:
Ah and I guess my comment doesn't apply since the new `continuous_eval` should be a drop-in replacement, right ?

Reply:
Yes, it's a drop-in replacement (unless someone uses `@continuous_eval` or relies on forcing defeq).